### PR TITLE
Beef up the README

### PR DESCRIPTION
**Pull request by taylortrimble**
Addresses #39, minus pydoc.

### CODE REVIEW

**Comment by mplewis** [comment]
Yay MIT license! :100: 

Should I work on pydoc and integrate into this PR or put it in its own?

**Comment by taylortrimble** [comment]
We can resurrect #160 and do it there, I think.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,129 @@
...
+
+<!-- TODO: Discuss Tesla Works voice and messaging -->
+
+To update project data, information about project leaders, and special redirects, you'll currently need to directly update the database. Layout of the database can be gleaned from [`external/postgres/create-tables.sql`](external/postgres/create-tables.sql).
+
+Photo dimensions were chosen to look best on Retina displays. Be aware of maximum image sizings for different screen sizes! Our projects grid on the home page is widest on mobile devices, since the grid becomes a vertical list.
+
+| Photo Dimensions          | Width       | Height      |
+|---------------------------|-------------|-------------|
+| Home page jumbotron       | ???         | ???         |
```
Comment (mplewis) [inline README.md:73]: Is this a thing?

**Comment by mplewis** [comment]
Once that comment is addressed, :shipit:

File: README.md (hunk 1 above)
Comment (taylortrimble) [inline README.md:73]: That's our TW logo, but that I want to change vewy soon.

**Pull request merged**